docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -932,7 +932,8 @@ This command provides password related actions.
 
 #### hash
 
-This command generates the hash of a given password.
+This command generates the hash of a given password. The hash is specified in the [credentials file](../JSON-RPC-API/Authentication.md#credentials-file)
```

Comment:
s/The hash is specified/This hash is what should be included/

Reply:
Updated to be active voice